dotnet/src/SemanticKernel.Abstractions/SkillDefinition/FunctionsView.cs

```diff
@@ -17,19 +18,21 @@ namespace Microsoft.SemanticKernel.SkillDefinition;
 [DebuggerDisplay("{DebuggerDisplay,nq}")]
 public sealed class FunctionsView
 {
+    private object _lock = new();
```

Comment:
Small formatting improvement:
```suggestion
    private object _lock = new();
    
```